adapters/to_sql.py

```diff
@@ -0,0 +1,120 @@
+"""Converts the request arguments to a SQL query."""
+import datetime
+from resources.dag_name_parser import dag_name_parser
+from data.state import State
+
+
+def to_sql_releases(filter_options):
+  """Converts a set of parameters contained in 'args' to a SQL query.
+
+  Gets the releases(AKA dag runs) which fit the parameters.
+
+  Args:
+    filter_options: object with filtering parameters
+
+  Returns:
+    sql_query: a string which is dynamically constructed using all of the params
+  """
+  sql_query = 'SELECT dag_id, execution_date FROM dag_run'
+  # convert start and end date from unix to python datetime
+  start_date = datetime.datetime.fromtimestamp(filter_options.start_date)
+  end_date = datetime.datetime.fromtimestamp(filter_options.end_date)
+  # append the date filter to the query based on:
+  # datetype, start_date, and end_date
+  if filter_options.datetype == 0:
```

Comment:
it is the same outcome if datetype == 0 or not. Is this expected ?

Reply:
it might worthwhile to look at jinja for query templating as this might be easier to read.

Reply:
I think my original idea was to be able to filter by last modified, but since this is dependent on tasks this turns out to have no significance anymore. Deleting